hs_core/views/__init__.py

```diff
@@ -611,6 +611,17 @@ def delete_metadata_element(request, shortkey, element_name, element_id, *args,
     return HttpResponseRedirect(request.META['HTTP_REFERER'])
 
 
+def delete_author(request, shortkey, element_id, *args, **kwargs):
+    try:
+        res, _, _ = authorize(request, shortkey, needed_permission=ACTION_TO_AUTHORIZE.EDIT_RESOURCE)
+        res.metadata.delete_element('creator', element_id)
+        resource_modified(res, request.user, overwrite_bag=False)
+        ajax_response_data = {'status': 'success', 'message': "Author was deleted successfully"}
+    except Error as exp:
+        ajax_response_data = {'status': 'error', 'message': exp.message}
+    return JsonResponse(ajax_response_data)
```

Comment:
optional - consistency on single vs double quotes

Reply:
does .status map to 200 / 404, 500 etc? I'm guessing you're following existing patterns in HydroShare but might be an opportunity to fix something if this is non-standard. https://docs.djangoproject.com/en/1.11/ref/request-response/#httpresponse-subclasses

Reply:
Implications across the codebase might be too large, anyway. Maybe just open a GH Issue if you find this is a bad practice across the repo

Reply:
This property does not map to response status. It is simply used in the frontend to assert if the request was performed successfully or not.